src/openai/constants/constants.ts

```diff
@@ -1,12 +1,33 @@
 import * as fs from 'fs';
-// export type ConversationMessages = Record<string, string>;
-// change back to this type, infer the conversation from backend and format into objects of messages to the api
-const messages = fs.readFileSync('src/openai/lib/data/messages.txt', 'utf8');
-export const MESSAGES: string = messages;
+const exampleConversation = fs.readFileSync('src/openai/lib/data/messages.txt', 'utf8');
```

Comment:
change 'src/openai/lib/data/messages.txt' url to const and also 'utf8' to a const

Reply:
done